aten/src/ATen/cpu/vec/functional_base.h

```diff
@@ -239,6 +239,24 @@ inline void map(
   }
 }
 
+template <typename scalar_t, typename Op>
+inline void map(
+    const Op& vec_fun,
+    std::vector<scalar_t>& output_data,
+    const std::vector<scalar_t>& input_data,
+    int64_t size) {
+  using Vec = vec::Vectorized<scalar_t>;
+  int64_t d = 0;
+  for (; d < size - (size % Vec::size()); d += Vec::size()) {
+    Vec output_vec = vec_fun(Vec::loadu(input_data.data() + d));
```

Comment:
Why not just call `.data()` on the vectors at the call site?

Reply:
Oops, that's indeed better! Thank you!